bench/commands/utils.py

```diff
@@ -111,13 +111,14 @@ def backup_all_sites():
 
 
 @click.command('release')
-@click.argument('app', type=click.Choice(['frappe', 'erpnext', 'erpnext_shopify', 'paypal_integration', 'schools']))
+@click.argument('app')
 @click.argument('bump-type', type=click.Choice(['major', 'minor', 'patch']))
+@click.argument('owner', default='frappe')
```

Comment:
The right term here should be `fork` instead of `owner`

i.e. `bench release erpnext patch --fork=revant`